src/appComponents/tracks/TracksController.js

```diff
@@ -1,100 +1,119 @@
-    $stateProvider.state('tracks', {
-        url: '/tracks',
-        templateUrl: 'appComponents/tracks/tracks.html',
-        controller: 'TracksController'
-    });
 }]);
 
-tracksModule.controller('TracksController', 
-	['$mdDialog','$sessionStorage', '$rootScope', 'ApiJsonFactory',
-        function($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
-		var tc = this;
-        if ( $sessionStorage.tracks === null ||
-            typeof($sessionStorage.tracks) == 'undefined')
-        {
+tracksModule.controller("TracksController", ["$mdDialog","$sessionStorage", "$rootScope", "ApiJsonFactory",
+        function ($mdDialog, $sessionStorage, $rootScope, ApiJsonFactory) {
+          var tsc = this;
+
+          if ($sessionStorage.tracks === null || 
+            typeof $sessionStorage.tracks === "undefined")
+         {
+
             $sessionStorage.tracks = [];
         }
-        tc.Tracks = $sessionStorage.tracks;
-        tc.Sessions = $sessionStorage.sessions;
+        tsc.Tracks = $sessionStorage.tracks;
+        tsc.Sessions = $sessionStorage.sessions;
         
+        if (tsc.Tracks.length === 0) {
 
-        if (tc.Tracks.length === 0) {
-            ApiJsonFactory.getJson('tracks')
-                .then(function (response) {
-                    tc.Tracks = response.data.tracks;
-                    $sessionStorage.tracks = tc.Tracks;
-                }, function (error) {
-                    console.error(error);
-                });
+          ApiJsonFactory.getJson("tracks")
+            .then(function (response) {
+
+            tsc.Tracks = response.data.tracks;
+            $sessionStorage.tracks = tsc.Tracks;
+                
+            }, function error (error) {
+                    //console.error(error);
+            });
 
-            ApiJsonFactory.getJson('sessions')
-                .then(function (response) {
-                    tc.Sessions = response.data.sessions;
-                    $sessionStorage.sessions = tc.Sessions;
-                }, function (error) {
-                    console.error(error);
+          ApiJsonFactory.getJson("sessions")
+            .then(function (response) {
+
+            tsc.Sessions = response.data.sessions;
+            $sessionStorage.sessions = tsc.Sessions;
+
+            }, function (error) {
+                    //console.error(error);
                 });
+
         }
 
-        tc.showSession = function(track, event) {
-            //singleTrack = track;
-            $mdDialog.track = {
-                singleTrack: track
-            };
-            $mdDialog.show({
-                controller: 'TrackDialogController',
-                templateUrl: 'appComponents/tracks/trackdialog.html',
-                parent: angular.element(document.body),
-                targetEvent: event,
+        tsc.showSession = function(track, event) {
+            // singleTrack = track;
+          $mdDialog.track = {
+                "singleTrack": track
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 space characters but found 16.](https://www.codacy.com/app/triplez/open-event-webapp/file/2418442828/issues/source?bid=3233857&fileBranchId=3237740#l58)